http_examples.cpp

```diff
@@ -131,6 +131,28 @@ int main() {
     work_thread.detach();
   };
 
+  // GET-example for the path /parse. Shows how to parse the parameters in the GET request. Responds firstName+" "+lastName
+  // For instance GET /parse?name=Mary&lastName=Smith will receive: Mary Smith
+  server.resource["^/parse$"]["GET"] = [](shared_ptr<HttpServer::Response> response, shared_ptr<HttpServer::Request> request) {
+    try {
+      SimpleWeb::CaseInsensitiveMultimap query = request->parse_query_string();
+
+      if(query.find("name") == query.end() || query.find("lastName") == query.end()) {
+        throw std::invalid_argument("Invalid or missing parameters in request");
+      }
+
+      auto name = query.find("name")->second +" "+ query.find("lastName")->second;
+
+      *response << "HTTP/1.1 200 OK\r\n"
+                << "Content-Length: " << name.length() << "\r\n\r\n"
+                << name;
+    }
+    catch(const exception &e) {
+      *response << "http/1.1 400 bad request\r\ncontent-length: " << strlen(e.what()) << "\r\n\r\n"
+                << e.what();
+    }
+  };
+
```

Comment:
One idea could be to instead add the query string parsing to the `/info` request example.

Reply:
Yeah, sure. Feel free to modify it however you want. Adding it to /info seems like a good idea. 
The important thing is to show in any of the examples the use of parse_query_string() so people will know it exists and how to use it.